README.md

```diff
@@ -67,6 +107,13 @@ This example requires a few arguments to be passes from command line. Chack them
 
 `julia  --project=. --color=yes "performance_benchmarks.jl" --help`
 
+3. For the `inverse lighting demos` we need to have the `tree.mtl` and `tree.obj` files.
```

Comment:
Maybe add a few words to describe each file? (Say, "material description" and "3D model"?)